setup.cfg

```diff
@@ -32,17 +32,17 @@ python_requires = >=3.9,<3.12
 setup_requires = setuptools_scm
 include_package_data = True
 install_requires =
-    numpy>=1.21
+    numpy>=1.22
     astropy>=4.3
     extension-helpers>=0.1
     packaging>=0.19
     scipy>=1.7
-    matplotlib>=3.3
+    matplotlib>=3.7
     PyYAML>=5.1
     configobj>=5.0.6
     scikit-learn>=1.0
     IPython>=7.10.0
-    ginga>=3.2
+    ginga>=4.1.1
     linetools
     qtpy>=1.9
```

Comment:
Don't know if it matters, but `ginga` requires `qtpy>=2.0.1` (released Feb 2022) for compatibility with PyQt6.

Reply:
Thanks for catching this!  I'll bump this, too.